django_mongodb/lookups.py

```diff
@@ -24,11 +24,28 @@ def is_null(self, compiler, connection):
     return connection.mongo_operators["isnull"](lhs_mql, self.rhs)
 
 
+# from https://www.pcre.org/current/doc/html/pcre2pattern.html#SEC4
+REGEX_MATCH_ESCAPE_CHARS = (
+    ("\\", r"\\"),  # general escape character
+    ("^", r"\^"),  # start of string
+    ({"$literal": "$"}, r"\$"),  # end of string
+    (".", r"\."),  # match any character
+    ("[", r"\["),  # start character class definition
+    ("|", r"\|"),  # start of alternative branch
+    ("(", r"\("),  # start group or control verb
+    (")", r"\)"),  # end group or control verb
+    ("*", r"\*"),  #  0 or more quantifier
+    ("+", r"\+"),  #  1 or more quantifier
+    ("?", r"\?"),  # 0 or 1 quantifier
+    ("{", r"\}"),  # start min/max quantifier
+)
```

Comment:
So this will trigger 13 `$replaceAll` when doing lookups?

Reply:
is there a way to do that replacements in python? and pass the final string

Reply:
When the value to be matched is a string, `re.escape()` [handles it](https://github.com/mongodb-labs/django-mongodb/blob/5f2a094293636f6d624f4b258f155c8fd01b46d2/django_mongodb/operations.py#L111-L113). This code is only for the case when `regexMatch`'s `regex` is a column.